_data/2020_checker.yml

```diff
@@ -48,12 +56,30 @@ en:
                 <li>Administrators - please see <a href="#action-authoritative-dns-operators">description for DNS admins</a>
                         and correct problems reported in the technical report below.</li>
             </ul>
+    clientReportFail:
+        short: "Fatal errors detected!"
+        long: >
+            <ul>
+                <li><strong>Your computer will have problems accessing some sites after DNS flag day 2020!</strong></li>
+                <li>Please retry the test to eliminate possibility of random network failures.</li>
+                <li>Users - please contact your computer administrator and point them to this page.</li>
+                <li>Administrators - this system is not able to complete DNS resolution for domains which depend on working DNS-over-TCP protocol.
+                    Please see <a href="#action-dns-resolver-operators">description for DNS admins</a> and correct problems.</li>
+            </ul>
+    reportNetworkProblem:
+        short: "Network problem!"
+        long: >
+            <ul>
+                <li><strong>The test could not be completed due to a network problem!</strong></li>
+                <li>This can also happen if Check My DNS is down or temporary unavailable, please check <a href="https://cmdns.dev.dns-oarc.net">https://cmdns.dev.dns-oarc.net</a> before trying again.</li>
```

Comment:
```suggestion
                <li>This can also happen if Check My DNS service is down or temporary unavailable, please check <a href="https://cmdns.dev.dns-oarc.net">https://cmdns.dev.dns-oarc.net</a> before trying again.</li>
```